Extensions/ArtifactEngine/Providers/filesystemProvider.ts

```diff
@@ -19,8 +19,8 @@ export class FilesystemProvider implements models.IArtifactProvider {
         return this.getItems(itemsPath, artifactItem.path);
     }
 
-    getArtifactItem(artifactItem: models.ArtifactItem): Promise<Stream.Readable> {
-        var promise = new Promise<Stream.Readable>(async (resolve, reject) => {
+    getArtifactItem(artifactItem: models.ArtifactItem): Promise<NodeJS.ReadableStream> {
```

Comment:
what is this change for?

Reply:
After applying unzip transformation what we get is a ReadableStream which is the base call for Stream.Readable, we need to change the signatures to accommodate for that.